src/divider/Divider.js

```diff
@@ -1,21 +1,27 @@
 import React from 'react';
-import { View, StyleSheet } from 'react-native';
-import colors from '../config/colors';
+import { View, StyleSheet, Platform } from 'react-native';
 import ViewPropTypes from '../config/ViewPropTypes';
 
 let styles = {};
 
-const Divider = ({ style }) =>
-  <View style={[styles.container, style && style]} />;
+const Divider = ({ style }) => <View style={[styles.container, style]} />;
 
 Divider.propTypes = {
   style: ViewPropTypes.style,
 };
 
 styles = StyleSheet.create({
   container: {
-    height: 1,
-    backgroundColor: colors.grey5,
+    ...Platform.select({
+      ios: {
+        height: 0.5,
```

Comment:
Should not the `height` be `StyleSheet.hairlineWidth`?
https://facebook.github.io/react-native/docs/stylesheet.html#hairlinewidth